timemachine/cpp/src/neighborlist.hpp

```diff
@@ -62,7 +62,7 @@ template <typename RealType> class Neighborlist {
     unsigned int *get_ixn_count() { return d_ixn_count_; }
 
     // get max number of row blocks
-    int B() const;
+    int row_blocks() const;
```

Comment:
can we rename this to `num_row_blocks()`?

Reply:
Should we also rename `column_blocks` to `num_column_blocks`?

Reply:
yes!